docs-source/content/faq/boot-identity-not-valid.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "Boot Identity Not Valid"
+date: 2020-03-02T08:08:19-04:00
+draft: false
+weight: 1
+---
+
+> One or more WebLogic Server instances in my domain will not start and I see errors in the server log like this:
+>
+> `<Feb 6, 2020 12:05:35,550 AM GMT> <Critical> <Security> <BEA-090402> <Authentication denied: Boot identity not valid. The user name or password or both from the boot identity file (boot.properties) is not valid. The boot identity may have been changed since the boot identity file was created. Please edit and update the boot identity file with the proper values of username and password. The first time the updatedboot identity file is used to start the server, these new values are encrypted.>`
+
```

Comment:
Theoretically, one won't see the above error if it came about because the domain secret changed.   If the domain secret changed, a different error is generated first and  the operator won't even attempt to start WLS.  The other error is: " trace SEVERE "Domain secret mismatch. The domain secret in 'DOMAIN_HOME/security/SerializedSystemIni.dat' where DOMAIN_HOME='$DOMAIN_HOME' does not match the domain secret found by the introspector job. WebLogic requires that all WebLogic servers in the same domain share the same domain secret. See 'Boot Identity Not Valid' in the operator FAQ (https://oracle.github.io/weblogic-kubernetes-operator/faq/boot-identity-not-valid/). MD5 checksum diff:"
"...

Reply:
Doesn't the fact that the server startup detecting the MD5 checksum has changed now change some of the FAQ content?

In other words, if you see the MD5 checksum error that means do X but if you still see Authentication Denied then do Y?

Reply:
Reformated to two different FAQ entries.